util/Migrator/DbMigrator.cs

```diff
@@ -12,39 +12,34 @@ public class DbMigrator
 {
     private readonly string _connectionString;
     private readonly ILogger<DbMigrator> _logger;
-    private readonly string _masterConnectionString;
 
-    public DbMigrator(string connectionString, ILogger<DbMigrator> logger)
```

Comment:
ℹ️ The logger is now created internally. Three callsites with no customization and two without DI.

Reply:
`SqlServerDbMigrator` got theirs from DI which could be customized, how about leave a constructor that takes it but offer one without which will just create the default one?

Reply:
Lemme know if https://github.com/bitwarden/server/pull/3910/commits/7d1eee7ae3a3f8ac5c45ce73263e86eeb60a5f37 works for you.

Reply:
Perfect